demo.py

```diff
@@ -0,0 +1,60 @@
+import os
+import argparse
+import shutil
+import sys
+from subprocess import call
+from types import SimpleNamespace
+from random import randint
+import gradio as gr
+import run_segmentation
+import run_monodepth
+from PIL import Image
+
+def run_cmd(command):
+    try:
+        print(command)
+        call(command, shell=True)
+    except KeyboardInterrupt:
+        print("Process interrupted")
+        sys.exit(1)
+
+run_cmd("wget --continue https://github.com/intel-isl/DPT/releases/download/1_0/dpt_large-midas-2f21e586.pt -O weights/dpt_large-midas-2f21e586.pt")
+run_cmd("wget --continue https://github.com/intel-isl/DPT/releases/download/1_0/dpt_large-ade20k-b12dca68.pt -O weights/dpt_large-ade20k-b12dca68.pt")
+
+def process(img, inference_type):
+    _id = randint(1, 10000)
+    INPUT_DIR = "/tmp/input_image" + str(_id) + "/"
+    OUTPUT_DIR = "/tmp/output_image" + str(_id) + "/"
+    run_cmd("rm -rf " + INPUT_DIR)
```

Comment:
Executing `rm -rf` with paths that are constructed by a wep-app is very dangerous. I can't merge this with shell commands that are excecuted.